src/debug/createdump/crashinfo.cpp

```diff
@@ -493,11 +493,11 @@ void
 CrashInfo::InsertMemoryRegion(uint64_t address, size_t size)
 {
     // Round to page boundary
-    uint64_t start = address & PAGE_MASK;
+    uint64_t start = ROUND_DOWN_TO_PAGE(address);
```

Comment:
@janvorli  The changes to the files in this directory are causing the Ubuntu x64 failures.  I had removed all references to PAGE_SIZE and PAGE_MASK from the source tree.  Since these ROUND_*_TO_PAGE macros were defined next to them I assumed they were safe to use....

In any case these macros must be coming indirectly from the host include files. 

Reverting all the changes in this directory allows Ubuntu compile.

My plan is to add these to the list of fixes in the SOS debugger issue.